crates/network-scheduler/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "network-scheduler"
-version = "1.2.0"
+version = "1.2.1"
```

Comment:
`Cargo.lock` version should also have been updated. Try to run `cargo fetch` to sync

Reply:
done